pubspec.yaml

```diff
@@ -5,13 +5,13 @@ description: >-
 homepage: https://github.com/dart-lang/json_rpc_2
 
 environment:
-  sdk: ">=2.2.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  stack_trace: ^1.0.0
+  stack_trace: ^1.10.0
   stream_channel: ">=1.1.0 <3.0.0"
 
 dev_dependencies:
-  pedantic: ^1.8.0
-  test: ^1.0.0
-  web_socket_channel: ^1.1.0
+  pedantic: ^1.11.0
+  test: ^1.16.8
```

Comment:
@natebosch just for my own education, what was the purpose of this nit?

Reply:
We have a general preference to only pin to a patch version when there is a known bug that was impacting the package, otherwise we keep to the oldest feature version that is compatible.

It's less impactful for `dev_dependencies`, but it's a general habit.